Sources/Rendering/OpenGL/RenderWindow/index.js

```diff
@@ -380,6 +380,8 @@ function vtkOpenGLRenderWindow(publicAPI, model) {
 
     camera.setPhysicalScale(physicalScale);
     camera.setPhysicalTranslation(physicalTranslation);
+    // Clip at 0.1m, 100.0m in physical space by default
+    camera.setClippingRange(0.1 * physicalScale, 100.0 * physicalScale);
```

Comment:
Worth throwing in docs somewhere?

Reply:
Probably not yet as XR clipping behavior is still buggy. I've opened #2452 to track.